tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md

```diff
@@ -0,0 +1,155 @@
+- Jika kamu tidak terkoneksi ke internet, kamu akan ditanyakan untuk memilih jaringan nirkabel, jika tersedia. Kami sarankan agar terkoneksi ketika melakukan pemasangan jadi kami bisa memastikan mesin mutakhir.
+
+
+![screenshot](https://assets.ubuntu.com/v1/3bbb0e35-download-desktop-install-ubuntu-desktop_2.jpg)
+
+## Alokasi ruang drive
+Duration: 0:01
+
+Gunakan kotak centang untuk memilih apakah kamu ingin Memasang Ubuntu berdampingan dengan sistem operasi lain, menghapus sistem operasi yang Anda dan menggantikannya dengan Ubuntu, atau — jika kamu adalah pengguna tingkat lanjut  — pilih opsi  ’**Something else**’.
+
+![screenshot](https://assets.ubuntu.com/v1/e07b98b8-ubuntu-installation-type.png)
+
+positif
+: Ubuntu membutuhkan 4.5 GB ruang untuk pemasangan minimal. Tetapi kamu akan membutuhkan ruang untuk data dan tambahan aplikasi yang kamu pasang.
+positive
+
+## Memulai Instalasi
+Duration: 0:01
+
+Setelah konfigurasi penyimpanan, klik tombol 'Pasang Sekarang'. Sebuah panel kecil akan muncul dengan rangkuman tentang opsi penyimpanan yang telah kamu pilih, untuk menetapkan perubahan.
+
+Klik `Lanjutkan` untuk membenarkan perubahan di tempat dan memulai proses installasi.
+
+![screenshot](https://assets.ubuntu.com/v1/a76b68ce-ubuntu-verify-install.png)
+
+## Pilih lokasi kamu
+Duration: 0:01
+
+Jika kamu terkoneksi dengan internet, lokasi kamu akan terdeteksi otomatis. Cek lokasi kamu jika benar dan klik 'Meneruskan' untuk memproses.
+
+Jika kamu tidak yakin dengan zona waktu kamu, tulis nama kota lokal atau kota atau gunakan peta untuk memilih lokasi.
+
+![screenshot](https://assets.ubuntu.com/v1/a942aa3d-download-desktop-install-ubuntu-desktop_6.jpg)
+
+positif
+: Jika kamu memiliki masalah mengkoneksikan ke Intenet, gunakan menu di pojok kanan atas sudut untuk memilih jaringan.
+
+## Pilih preferensi tata letak keyboard
+Duration: 0:01
+
+Untuk memberitahu Ubuntu tentang tata letak papan ketik kamu, klik pada opsi bahasa yang kamu perlukan, diikuti dengan tata letak spesifik papan ketik kamu. Jika kamu tidak yakin, klik tombol '**Deteksi Tata Letak Keyboard**' untuk bantuan.
+
+Tata letak keyboard bisa selalu berubah, dan tata letak tambahan ditambahkan, dari desktop setelah instalasi.
+
+![screenshot](https://assets.ubuntu.com/v1/310b5196-download-desktop-install-ubuntu-desktop_7.jpg)
+
+## Informasi Login
+Duration: 0:01
+
+Masukkan nama kamu dan pemasang akan otomatis mensugestikan nama komputer dan username. Ini bisa berubah dengan mudah jika kamu lebih suka. Nama komputer adalah bagaimana komputer akan muncul di jaringan, Username kamu akan menjadi login kamu dan nama akun.
+
+Selanjutnya, masukkan password yang kuat. Pemasang akan memberitahu jika itu sangat lemah.
+
+Kamu bisa memilih otomatis login dan enkripsi folder rumah. Jika mesin kamu portable, kami merekomendasikan tetap login otomatis dimatikan atau menyalakan enskripsi. Ini akan mencegah orang lain mengakses file personal kamu jika mesin hilang atau dicuri.
```

Comment:
we recommend keeping automatic login disabled -> kami merekomendasikan agar login otomatis dimatikan